docker-run-server.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+python manage.py migrate
+python manage.py loaddata sites
+honcho start web
```

Comment:
Make sure to use `exec honcho start web` to ensure honcho and gunicorn correctly receive signals.